src/cascadia/WindowsTerminal/WindowsTerminal.vcxproj

```diff
@@ -120,13 +120,6 @@
     <ReasonablePlatform Condition="'$(Platform)'=='Win32'">x86</ReasonablePlatform>
     <ReasonablePlatform Condition="'$(ReasonablePlatform)'==''">$(Platform)</ReasonablePlatform>
   </PropertyGroup>
-
-  <ItemGroup Condition="'$(WindowsTerminalOfficialBuild)'=='true'">
-    <!-- Add all the CRT libs as content -->
-    <_OpenConsoleVCLibToCopy Include="$(VCToolsRedistInstallDir)\$(ReasonablePlatform)\Microsoft.VC142.CRT\*.dll">
-      <TargetPath>%(Filename)%(Extension)</TargetPath>
-    </_OpenConsoleVCLibToCopy>
-  </ItemGroup>
   <!-- **END VC LIBS HACK** -->

```

Comment:
Should this comment still be here since the hack was moved down?

Reply:
There are three regions impacted by the hack, maybe I can consolidate them.